### PR TITLE
Fix spelling errors in Google.Cloud.Diagnostics.* Libs

### PR DESCRIPTION
**Pull request by iantalarico**

### CODE REVIEW

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/CommonUtilsTest.cs [hunk 1]
```diff
@@ -42,16 +42,16 @@ public void GetAndCheckProjectId()
             };
 
             // The string project id is not null and the MonitoredResource contains a project id
-            // so the string project id is retruned.  We do this as the explictly set project id should be
+            // so the string project id is returned.  We do this as the explicitly set project id should be
             // defaulted too.
```
Comment (evildour) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/CommonUtilsTest.cs]: "too" => "to"

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Trace/TraceUtils.cs [hunk 2]
```diff
@@ -51,7 +51,7 @@ internal static RateLimiter GetRateLimiter(double qps, long[] elapsedMillisecond
         }
 
         /// <summary>
-        /// Checks if a span's labels are are equal to a dictionary strings.
+        /// Checks if a span's labels are equal to a dictionary strings.
```
Comment (evildour) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Trace/TraceUtils.cs]: Should this be "a dictionary _of_ strings"?

**Comment by iantalarico** [comment]
All fixed, PTAL.

**Comment by iantalarico** [comment]
Thanks Mike!

**Pull request merged**